src/Compilers/Core/Portable/Symbols/ISymbolInternal.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
@RikkiGibson ran into some cases where DeclaringCompilation was null, I think.

Reply:
> ran into some cases where DeclaringCompilation was null

I omitted this property for now, similar to the `Containing*` properties in #39180.

Reply:
I now made this explicit.